src/components/organisms/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -0,0 +1,384 @@
+<template>
+  <section
+    v-if="!isLoading"
```

Comment:
R: IMO it shouldn't be here. To be honest this component should be split into the smallest parts.
IMO wee need split it to 
* country selector to get phone codes and it should handle state where we waiting for codes 
* phone number input